ios/RNZcash.swift

```diff
@@ -155,6 +155,19 @@ class RNZcash : RCTEventEmitter {
         }
     }
 
+    // A convenience method to get the block height when the synchronizer isn't running
+    @objc func getBirthdayHeight(_ host: String, _ port: Int, resolver resolve:@escaping RCTPromiseResolveBlock, rejecter reject: RCTPromiseRejectBlock) {
+        do {
+            let endpoint = LightWalletEndpoint(address: host, port: port, secure: false)
```

Comment:
Does setting `secure: false` mean we aren't using SSL? I guess that is OK, since we are just grabbing the current height.

Reply:
That was a bug! Should be true